### ISSUE TITLE
betterdiscord addon error and crashing when clicking for details

### ISSUE THREAD

**Issue by TheDutchGinger** [BetterDiscord/BetterDiscord#1229]
![image](https://user-images.githubusercontent.com/99912480/154552663-6afe1687-291a-48a7-92ec-81835561cc5a.png)
(it's not letting me take a screen shot of the "discord crashed" screen but it says :well, this is awkward looks like discord crashed unexpectedly  we've tracked the errors and will get right on it)

**Comment by CodeKid0** [comment]
got the same thing literally everytime i tried to right-click on a server. reinstall the entire discord and it should work

**Comment by TheDutchGinger** [comment]
I have tried to fully reinstall tried reinstalling with the setup program and tried repairing it but it doesnt help

**Comment by Matt0040** [comment]
I was able to get it back working, however I don't want to try to replicate it due to the fragility caused by the update. I removed the "update" file in discord's main folder, then placed it somewhere else. Next I opened the "app-1.0.9003" folder (the previous version of discord) and used the discord application within to open discord. I'm unsure if this will work for long, but it is working at the moment.
